models/src/main/protobuf/RhoTypes.proto

```diff
@@ -129,12 +167,12 @@ message ESet {
 }
 
 message EMap {
-    repeated ParTuple kvs = 1;
+    repeated KeyValuePair kvs = 1;
```

Comment:
Leave this for now, but I want you to be thinking about how we can map these to collections.immutable.map
Similarly for Set.

Reply:
Sets are done. Will think about maps as mentioned in Discord.